packages/front-end/components/FactTables/FactMetricList.tsx

```diff
@@ -51,7 +51,7 @@ export default function FactMetricList({ factTable }: Props) {
     searchFields: ["name^3", "description"],
   });
 
-  const canEdit = permissions.check("createMetrics", factTable.projects || "");
+  const canCreateMetrics = permissionsUtil.canCreateMetric(factTable);
```

Comment:
Make this clear that you are passing in a metric-to-be-created and not a fact table:

```ts
permissionsUtil.canCreateMetric({ projects: factTable.projects });
```